CoreGraphFramework/src/au/gov/asd/tac/constellation/graph/locking/ValueStore.java

```diff
@@ -67,41 +67,41 @@ public void setObject(final Object value) {
     }
 
     public final void save() {
-        converter.save();
+        converter.save(access);
     }
 
-    private static interface Saver extends Serializable {
```

Comment:
I wonder what performance/memory complications there are when we remove the `static` modifier. @formalhaut69 did this have to be done to make it a lambda or was it for another reason?

Reply:
> https://gitter.im/constellation-app/community?at=5e83ebd9b6d72077bcabfe17

@cygnus-x-1 I've just caught up on the chat in Gitter. Can you explain why having the `static` was a bad design?

Reply:
> @cygnus-x-1 I've just caught up on the chat in Gitter. For my understanding, can you explain why having the `static` is bad design?

Having an inner static Interface like that is not inherently bad design (although it does imply it's supposed to be used outside this class); the bad design part was then having it rely on a variable of its outer class, as that variable will only exist when that outer class is instantiated.